spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -342,4 +352,31 @@ public AutoTimeProperties getAutotime() {
 
 	}
 
+	public static class System {
+
+		private final DiskSpace diskspace = new DiskSpace();
+
+		public DiskSpace getDiskspace() {
+			return this.diskspace;
+		}
+
+		public static class DiskSpace {
+
+			/**
+			 * Paths to report disk metrics for.
+			 */
+			private List<File> paths = Arrays.asList(new File("."));
```

Comment:
`management.metrics.diskspace.path` or  `management.metrics.diskspace.paths`? 

Of course, the singular reads better when there is a single path. 
```yaml
management.metrics.diskspace.path: path1
```

And the plural reads better when there are multiple paths.
```yaml
management.metrics.diskspace.paths:
  - path1
  - path2
```

Reply:
That should be `paths`,  I think with "Comma-separated list of paths..." for the description.